### PR TITLE
feat(Charts): modified the Customize your charts section

### PR DESCRIPTION
**Pull request by nbaenam**
Modified the _Customize your charts_ section on the [Use your charts](https://docs.newrelic.com/docs/query-your-data/explore-query-data/use-charts/use-your-charts/) page. 

See [NR-61710](https://issues.newrelic.com/browse/NR-61710).

### CODE REVIEW

File: src/content/docs/query-your-data/explore-query-data/use-charts/use-your-charts.mdx [hunk 1]
```diff
@@ -13,101 +13,127 @@ redirects:
...
+import queriesnrqlCustomChartOthers from 'images/queries-nrql_screenshot-full_custom-chart-others.png'
 
 Once you've created a chart, you can customize the appearance of it to best present the data. You can share a chart in different formats and add a chart to a new or existing dashboard.
 
 ## Change the appearance of your chart [#change-appearance]
 
 When you run your query in NRQL mode or view your chart while using the data explorer to specify data, New Relic analyzes your data and applies a [chart type](/docs/insights/use-insights-ui/manage-dashboards/insights-chart-types) that fits your data.
 
-For some queries, you'll have several options of chart types to choose from. To change chart type, use the **Chart type** menu to the right of the current chart. Each type in the list has a tooltip with information about using that type.
+You've several options of chart types to choose from. To change chart type, use the **Chart type** menu to the right of the current chart.
```
Comment (jeff-colucci) [inline src/content/docs/query-your-data/explore-query-data/use-charts/use-your-charts.mdx:34]: "You've" reads very strange here, I'd suggest changing to "There are"

File: src/content/docs/query-your-data/explore-query-data/use-charts/use-your-charts.mdx [hunk 2]
```diff
@@ -13,101 +13,127 @@ redirects:
...
 
 <figcaption>
-  Example of the chart type menu, showing a tooltip.
+**[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Query your data**: Example of the chart type menu.
 </figcaption>
 
 ## Customize your charts [#customize-charts]
 
-While we try our best to optimize how we display your data, sometimes you may have other needs. Depending on the chart type, additional customization options are available.
+While we try our best to optimize how we display your data, sometimes you may have other needs. Depending on the chart type, there are available additional customization options.
```
Comment (jeff-colucci) [inline src/content/docs/query-your-data/explore-query-data/use-charts/use-your-charts.mdx:48]: Suggest slightly modifying "there are available additional customization options" to "there are additional customization options available"

File: src/content/docs/query-your-data/explore-query-data/use-charts/use-your-charts.mdx [hunk 3]
```diff
@@ -13,101 +13,127 @@ redirects:
...
-  alt="customchart04_y_legend.png"
-  src={queriesnrqlCustomChartLegend}
-/>
+<Collapser
+    id="bar-charts-pie-charts-tables"
+    title="Bar charts, pie charts, and tables"
+  >
 
-### Remove the other groups facet [#custom-others]
+  When [faceting](/docs/query-your-data/explore-query-data/dashboards/filter-new-relic-one-dashboards-facets) on bar charts, pie charts, or tables, and if the number of faceting on queries is larger than 2,000, the **Other groups** facet aggregates the rest of facets. With this customization you can select whether to see **Other groups**, or remove it.
```
Comment (jeff-colucci) [inline src/content/docs/query-your-data/explore-query-data/use-charts/use-your-charts.mdx:118]: I think this sentence could be improved slightly by a small restructure. Something like "If the number of [facets](/docs/query-your-data/explore-query-data/dashboards/filter-new-relic-one-dashboards-facets) on a query is larger than 2,000 for bar charts, pie charts, or tables, the **Other groups** facet aggregates the rest of the facets." I notice we use the word "facet" 3 times in this sentence, but I wouldn't know a good alternative for any of them.

Also. the "remove it" at the end of the second sentence reads a little strange. I know it refers to the option (and thus, "it"), but could "it" be changed to "them"?

**Review by jeff-colucci — COMMENTED** [review]
Just a few small things I saw here and there in this review. Hope it is helpful feedback!

File: src/content/docs/query-your-data/explore-query-data/use-charts/use-your-charts.mdx (hunk 1 above)
Comment (nbaenam) [reply src/content/docs/query-your-data/explore-query-data/use-charts/use-your-charts.mdx:34]: Yes, it's better. Thanks!!

File: src/content/docs/query-your-data/explore-query-data/use-charts/use-your-charts.mdx (hunk 2 above)
Comment (nbaenam) [reply src/content/docs/query-your-data/explore-query-data/use-charts/use-your-charts.mdx:48]: Thank you! This one it's also much better ;-)

File: src/content/docs/query-your-data/explore-query-data/use-charts/use-your-charts.mdx (hunk 3 above)
Comment (nbaenam) [reply src/content/docs/query-your-data/explore-query-data/use-charts/use-your-charts.mdx:118]: I've modified the entire paragraph. Let me know if it improves or not. Thanks!

**Review by jeff-colucci — COMMENTED** [review]
Trying this again, looks good to me and should be all set to merge.

**Comment by nbaenam** [comment]
Thank you @jeff-colucci! I need your approval to merge ;-)

**Review by jeff-colucci — APPROVED** [review]
Reviewed and looks good

**Pull request merged**